### PR TITLE
Add static publishers

### PR DESCRIPTION
**Pull request by gwalck**
Mostly adding a way to publish tf through static publishers + tiny other things

### CODE REVIEW

File: transforms.py [hunk 1]
```diff
@@ -103,7 +104,7 @@ def __init__(self, node, tcp_link_name='tcp_link', tool_link_name='tcp_gripper')
         self.tcp_frame = tcp_link_name
         self.tool_frame = tool_link_name
 
-    def to_from_tcp_pose_conversion(self, pose_source_frame, source_frame, target_frame, apply_tool_offset=True):
+    def to_from_tcp_pose_conversion(self, pose_source_frame: Pose, source_frame: str, target_frame: str, apply_tool_offset=True) -> Pose:
```
Comment (destogl) [inline transforms.py]: Let's add to all methods types, since we are adding those already.

File: .gitignore
Comment (destogl) [inline .gitignore:1]: Please remove and feel free to use global one for your environment inside your "$HOME" folder.

Comment (gwalck) [reply .gitignore:1]: ok

File: transforms.py (hunk 1 above)
Comment (gwalck) [reply transforms.py]: indeed, I had the plan to do it and then forgot.

**Review by destogl — APPROVED** [review]

**Pull request merged**